<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormIntegral</class>
 <widget class="QWidget" name="FormIntegral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>420</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>510</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Численное интегрирование</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>f(x) =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFunction"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="8">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Ход выполнения итеррационного процесса</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditIterProcess">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Пределы интегрирования</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>a=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxA"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>b=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxB"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Требуемая погрешость расчета</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ε=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxEpsilon">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>0.001000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Количество десятичных знаков</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBoxPrec">
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" rowspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Выбор метода численного
интегрирования</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Количество итерраций</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>N</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditIterrationNumber"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QRadioButton" name="radioButtonLeftRect">
       <property name="text">
        <string>Метод левых прямоугольников</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QRadioButton" name="radioButtonRightRect">
       <property name="text">
        <string>Метод правых прямоугольников</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Результат интегрирования</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>I = </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditRes"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QRadioButton" name="radioButtonMidleRect">
       <property name="text">
        <string>Метод средних прямоугольников</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QRadioButton" name="radioButtonTrap">
       <property name="text">
        <string>Метод трапеций</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" rowspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonCalculate">
         <property name="text">
          <string>Расчет</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Выйти</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <widget class="QRadioButton" name="radioButtonSimpson">
       <property name="text">
        <string>Метод Симпсона</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
